src/pages/Home/components/SupplyRank/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Chip } from '@/components';
+import { useSupplyRank } from '@/hooks/useSupplyRank/useSupplyRank';
+import { Loader } from 'lucide-react';
+
+export const SupplyRank = () => {
+  const { data: supplyRankRows, loading } = useSupplyRank();
+
+  return (
+    <div className="my-2">
+      <h2 className="text-[#2f2f2f] font-semibold text-xl pb-2">
+        Confira quais itens os abrigos mais necessitam:
+      </h2>
+      {loading ? (
+        <Loader className="justify-self-center self-center w-5 h-5 animate-spin" />
+      ) : (
+        <div className="flex gap-2 flex-wrap">
+          {supplyRankRows.map((rankRow, idx) => (
+            <Chip
+              className={'bg-light-red'}
+              key={idx}
```

Comment:
Em vários locais temos esse mesmo key sendo utilizado. Sugiro usar uma informação única para evitar mensagens no console.
```suggestion
              key={supply-rank-`${idx}`}
```

Reply:
O código está ai, fique a vontade para alterar